.github/workflows/build_run.yml

```
@@ -0,0 +1,32 @@
+name: Build Frontend, Backend  and Run
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: Build Images and Deploy Example WebApp
+    runs-on: ubuntu-16.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Test Yarn
+      uses:
+        node_version: '10.x'
+      run: |-
+        cd admin && \
+        yarn install && \
+        yarn export
+
+    - name: Test make
+      run: |-
+        cd .. && \
+        make build 
+
+    - name: Run
+      run: |-
+        openssl req -newkey rsa:2048 -nodes -keyout hetty_key.pem -x509 -days 365 -out hetty_cert.pem && \
+        ./hetty
```

As of https://github.com/dstotijn/hetty/commit/8b047478557755757fe7ced0d36e78b25c295dea, this shouldn't be needed anymore 🙌 
```suggestion
        ./hetty
```